services/cinema_hall.py

```diff
@@ -0,0 +1,19 @@
+from django.db.models import QuerySet
+
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> QuerySet:
+    return CinemaHall.objects.all()
+
+
+def create_cinema_hall(
+        hall_name: str,
+        hall_rows: int,
+        hall_seats_in_row: int
+) -> QuerySet:
```

Comment:
The return type for `create_cinema_hall` is currently specified as `QuerySet`, but the `create` method actually returns an instance of `CinemaHall`, not a `QuerySet`. Consider changing the return type to `CinemaHall`.